### PR TITLE
feat(tasks): implement assign, mark as done, cancel and resurrect actions

### PR DESCRIPTION
**Pull request by jnizet**

### CODE REVIEW

File: frontend/src/app/tasks-page/tasks-page.component.ts [hunk 1]
```diff
@@ -13,14 +17,37 @@ export class TasksPageComponent implements OnInit {
...
     this.route.data.subscribe(data => this.page = data.tasks);
   }
 
-  loadPage(pageNumber: number) {
-    this.router.navigate(['.'], {relativeTo: this.route, queryParams: {page: (pageNumber - 1).toString()}})
+  loadPage(viewPageNumber: number) {
+    this.router.navigate(['.'], {relativeTo: this.route, queryParams: {page: (viewPageNumber - 1).toString()}})
+  }
+
+  onTaskClicked(event: TaskEvent) {
```
Comment (cexbrayat) [inline frontend/src/app/tasks-page/tasks-page.component.ts]: I don't really get this method, a small comment would be welcomed

**Comment by cexbrayat** [comment]
Follow up: I implemented what I had in mind in #90

File: frontend/src/app/tasks-page/tasks-page.component.ts (hunk 1 above)
Comment (jnizet) [inline frontend/src/app/tasks-page/tasks-page.component.ts]: I'm all for clarifying with comments, or even better: a better name and comments. Should I rename it "onTaskActioned", or "onTaskAction", or "onTaskEvent"? What would you like to see as comment? Something like

> Method called from the template when a TaskEvent is emitted by the child TasksComponent, indicating that one of the action buttons (assign, edit, cancel, etc.) of a task has been clicked.

**Comment by jnizet** [comment]
FYI: don't start implementing task edition. I'm on it.

**Pull request merged**